### ISSUE TITLE
Ubuntu + Powershell 7 not finding LocalizedData

### ISSUE THREAD

**Issue by astrahan87** [PlagueHO/CosmosDB#332]
# Issue
Details:
- [ PowerShell 7.0.0 ] Version of PowerShell you're using
- [ ConsoleHost ] PowerShell host you're using
- [ Ubuntu 18.04.2 LTS ] Operating system you're running
- [ 3.5.2.487 ] Version of CosmosDB PowerShell Module you're using

You've fixed the "hard" errors that were occurring with Powershell 7, but we've started seeing some non-terminating error messages related to using Powershell 7:
```
>> Remove-CosmosDbCollection -Context $cosmosDbContext -Database $DatabaseId -Id $CollectionId
Write-Verbose: /usr/local/share/powershell/Modules/CosmosDB/3.5.2.487/CosmosDB.psm1:5110
Line |
5110 |  …    Write-Verbose -Message $($LocalizedData.NoResourceTokensInContext)
     |                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot bind argument to parameter 'Message' because it is null.
```
Just calling "Import-Module -Name CosmosDB" results in:
```
Import-LocalizedData: /usr/local/share/powershell/Modules/CosmosDB/3.5.2.487/CosmosDB.psm1:24
Line |
  24 | Import-LocalizedData `
     | ~~~~~~~~~~~~~~~~~~~~~~
     | 
Cannot find the PowerShell data file 'CosmosDB.strings.psd1' in directory '/usr/local/share/powershell/Modules/CosmosDB/3.5.2.487', or in any parent culture directories.
```

Seems like it's not recognizing to look in the /en-US/ subdirectory for some reason.

This was previously working with Powershell Core 6.2 (or at least erroring more quietly).
If this is a misconfiguration on our end for some reason, please let me know.

**Comment by topikettunen** [comment]
I'm having the same error when running `Import-Module -Name CosmosDB`. Currently I'm using this module in Azure Pipeline but for some reason it stopped working today. Yesterday everything seemed to be running all fine. Also when running locally there seems to be no problems. I have tested with the latest 3.5.2.487 version and also the second latest 3.5.1.458 but with no luck.

**Comment by PlagueHO** [comment]
Hi @astrahan87 and @topikettunen , I'm just working on updating the CI process to simplify build and release and will try and identify the cause of this problem at the same time.

@topikettunen - are you using Linux, Windows or MacOS agents?

**Comment by topikettunen** [comment]
I use various different agents but the job where I use this module runs Linux (Ubuntu 18.04) agent.

**Comment by PlagueHO** [comment]
Thanks @topikettunen - I'm currently only testing on Ubuntu 16.04. So I'll add some tests on Ubuntu 18.04 to see if I can catch/replicate this.

**Comment by PlagueHO** [comment]
I've added the Xenial tests and will wait for them to complete: https://dscottraynsford.visualstudio.com/GitHub/_build/results?buildId=1429&view=logs&s=859b8d9a-8fd6-5a5c-6f5e-f84f1990894e

It will take a while for the test runs to complete so I'll check on them tomorrow.

**Comment by PlagueHO** [comment]
@topikettunen - have replicated the issue. Looks like it only occurs on PS7 with Xenial and is to do with the case I was using to select the default culture. I've applied a fix and will aim to release tomorrow.

**Comment by PlagueHO** [comment]
Unfortunately the fix I applied didn't work. So I'm going to need to do some further investigation on this. You can see the test failure here:
https://dscottraynsford.visualstudio.com/GitHub/_build/results?buildId=1430&view=logs&j=1b4c76ef-e880-5241-0155-88566c4dcdab&t=c3b097e0-8d2d-5cef-f943-5ce972a892ac&l=81

It appears to be an issue occurring only on Ubuntu 18.04 on PS7 when using the `Import-LocalizedData` cmdlet. I'll need to set up an Ubuntu 18.04 machine to do some local debugging. I might need to implement a special "temporary" code path for Ubuntu 18.04 to handle this. It does seem like a bug in PS7, so I might also need to raise this with the PS team.

**Comment by PlagueHO** [comment]
I've found the problem: The `$PSUICulture` variable is empty on PS7 on Ubuntu 18.04. I've raised as an issue with the PS Team: https://github.com/PowerShell/PowerShell/issues/12149

However, I can work around the issue, so I've put a fix in place and just testing now: https://dev.azure.com/dscottraynsford/GitHub/_build/results?buildId=1432&view=results

**Comment by PlagueHO** [comment]
The tests have passed, so I'll aim to release this later today.

**Comment by PlagueHO** [comment]
This has been released to the PS Gallery in 3.6.1:
https://www.powershellgallery.com/packages/CosmosDB/3.6.1

**Comment by astrahan87** [comment]
Looks like it's all working for me. Thanks!
